client/src/plugins/version-info/ReleaseInfo.js

```diff
@@ -44,12 +44,19 @@ export function ReleaseInfo(props) {
     <div className={ css.ReleaseInfo }>
       <ul className="dashed">
         <li>
-          <h4>Edit FEEL expressions in the properties panel</h4>
-          This release adds syntax highlighting and autocomplete for FEEL expressions to the properties panel.
+          <h4>Support for extension properties in Camunda Platform 8</h4>
+          We now support generic extension properties, which will be available for execution starting from Camunda Platform 8.1.
         </li>
         <li>
-          <h4>Diagram errors in the properties panel</h4>
-          All diagram errors are shown in the properties panel as of this release. Open the error panel for the errors to be shown.
+          <h4>Diagram errors on the canvas</h4>
+          Diagram errors are now shown on the canvas. Click on an error to show details.
+        </li>
+        <li>
+          <h4>Sticky properties panel headers</h4>
+          The section headers of the BPMN and DMN properties panels are now sticky to provide reliable context.</li>
```

Comment:
...  to provide reliable context when you scroll.

Reply:
@philippfromme _reliable context_ means that users always now reliably in which group they are.